### PR TITLE
Update NLB guide to preserve IP address on EKS using AWS Load Balancer Controller strategy

### PR DESCRIPTION
**Pull request by nutellinoit**
### Summary
HI this PR address some outdated informations on the page `preserving client ip adress`. It adds informations on the AWS Load Balancer Controller instead of using the in-tree one.

The AWS Load Balancer Controller is a more complete approach on EKS to manage Load Balancers, network and application. 

### Reason
The reason behind this PR is to update the page and remove the wrong information that proxy protocol can be enabled on a `in-tree` `nlb` load balancer.

### Testing
To test the annotations/Load Balancers, you need an EKS cluster with the AWS Load Balancer Controller deployed.

To install the AWS Load Balancer Controller on an EKS cluster you can follow the official docs or use the kustomize package and terraform module included in this repository https://github.com/sighupio/fury-kubernetes-aws 

review:tech:

### CODE REVIEW

**Comment by Guaris** [comment]
@Kong/team-k8s Is this something that you can review?

**Review by rainest — APPROVED** [review]
We don't have an AWS test instance readily available to confirm, but on the face the changes look reasonable, so :+1: from me.

File: app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md [hunk 1]
```diff
@@ -84,28 +84,31 @@ You can use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
...
-  In this case, you need to use the Proxy Protocol method to preserve Client IP
-  address.
-- L7 Load Balancer
+You have three options:
+
+- L4 Network Load Balancer, with [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller): 
+  In this case, you can use the Proxy Protocol method to preserve Client IP
+  address together with `ExternalTrafficPolicy: Cluster`.
+- L4 in-tree Network Load Balancer with the `service.beta.kubernetes.io/aws-load-balancer-type: nlb` annotation: 
+  In this case you need to use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md:93]:
```suggestion
  In this case you need to use `ExternalTrafficPolicy: Local` to preserve the client IP address.
```

File: app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md [hunk 2]
```diff
@@ -84,28 +84,31 @@ You can use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
...
-You have two options:
-
-- L4 Load Balancer
-  In this case, you need to use the Proxy Protocol method to preserve Client IP
-  address.
-- L7 Load Balancer
+You have three options:
+
+- L4 Network Load Balancer, with [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller): 
+  In this case, you can use the Proxy Protocol method to preserve Client IP
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md:90]:
```suggestion
  In this case, you can use the Proxy Protocol method to preserve the client IP
```

File: app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md [hunk 3]
```diff
@@ -84,28 +84,31 @@ You can use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
...
+  address together with `ExternalTrafficPolicy: Cluster`.
+- L4 in-tree Network Load Balancer with the `service.beta.kubernetes.io/aws-load-balancer-type: nlb` annotation:
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md:92]:
```suggestion
- L4 in-tree network Load Balancer with the `service.beta.kubernetes.io/aws-load-balancer-type: nlb` annotation: 
```

File: app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md [hunk 4]
```diff
@@ -84,28 +84,31 @@ You can use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
...
-- L7 Load Balancer
+You have three options:
+
+- L4 Network Load Balancer, with [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller): 
+  In this case, you can use the Proxy Protocol method to preserve Client IP
+  address together with `ExternalTrafficPolicy: Cluster`.
+- L4 in-tree Network Load Balancer with the `service.beta.kubernetes.io/aws-load-balancer-type: nlb` annotation: 
+  In this case you need to use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
+- L7 Load Balancer: 
   In this case, you need to use the HTTP headers method to preserve the Client
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md:95]:
```suggestion
  In this case, you need to use the HTTP headers method to preserve the client
```

File: app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md [hunk 5]
```diff
@@ -84,28 +84,31 @@ You can use `ExternalTrafficPolicy: Local` to preserve the Client IP address.
...
   IP address.
 
-The recommend Load Balancer type for AWS is NLB.
-You can choose the type of Load Balancer using the following annotation:
+The recommended Load Balancer type for AWS is the Network Load Balancer deployed with AWS Load Balancer Controller.
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/preserve-client-ip.md:98]:
```suggestion
The recommended Load Balancer type for AWS is the Network Load Balancer deployed with the AWS Load Balancer Controller. 
```

**Review by Guaris — APPROVED** [review]

**Pull request merged**